.github/ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,4 @@
+<!-- Keep the progress checklist.  -->
 ## Progress
```

Comment:
Discussed with team this morning more about this issue. Current plan is to use the [GitHub project board](https://github.com/cerner/terra-core/projects/1) as status tracker. Ideally we find a way to automate adding labels to issues based on what column the issue card is in. Feel free to add the labels as we see fit, e.g. "Needs UX Review", "Need functional review" but we should be tracking this by having columns for these on our project board.

The board serves as a overall status of the project, while labels on an issue would serve as individual status when you don't want to view the project board. Both are valuable. However, I lean towards project board being more valuable than labels with the idea that we should automate labelling via the project board in the future.

> In response to who would track it, would it make sense that the individual who completed their relative piece update the progress status and move it to the next step?

Thats what I'm thinking, each core contributor manages their issues across the board. The core team should help manage this for outside contributions as well. I don't expect outside contributors to update our project board.

This process is not set in stone, but just a point we are at as we explore using GitHub to manage the project. If we find we want to add the Progress checklist back to the issue template, we can add it back.